reco_utils/dataset/pandas_df_utils.py

```diff
@@ -293,10 +290,8 @@ def negative_feedback_sampler(
         If the samples exceed the number of total possible negative feedback samples, it will be reduced to the number
         of all the possible samples.
         seed (int): seed for the random state of the sampling function.
-
```

Comment:
minor thing here, if we remove these intermediate lines, the sphinx documentation doesn't parse the docstring correctly

Reply:
hey @yueguoguo, since this is a small fix and you are on leave, I'll go ahead and fix it